### ISSUE TITLE
Matplotlib Deprecation Warning: imshow

### ISSUE THREAD

**Issue by pythonic2020** [spectralpython/spectral#118]
I intermittently get this warning 

```
C:\Users\me\Miniconda3\envs\py38\lib\site-packages\spectral\graphics\spypylab.py:27: MatplotlibDeprecationWarning: 
The keymap.all_axes rcparam was deprecated in Matplotlib 3.3 and will be removed two minor releases later.
  mpl.rcParams['keymap.all_axes'] = ''
```

after invoking imshow using this code in a Jupyter notebook, Python 3.8.6, matplotlib 3.3.3:
```
from spectral.graphics.spypylab import imshow 

view = imshow(data=cube1, bands=(76, 83, 92), origin='upper', stretch=(0.08, 0.98),
              source=cube1, title='Cube - Bands 76-83-92/RGB', figsize=(6, 6))
```
The warning does not effect imshow function as far as I can tell.

**Comment by cl1314** [comment]
have you solved this problem?

**Comment by tboggs** [comment]
@cl1314 yes it is fixed in GitLab. It will be included in the next release, probably in the next few days.
